packages/discovery-provider/src/api/v1/transactions.py

```diff
@@ -204,20 +213,21 @@ class GetUSDCTransactionHistory(Resource):
         params={"id": "A User ID"},
     )
     @full_user_ns.expect(usdc_transaction_history_parser)
-    @auth_middleware(usdc_transaction_history_parser)
+    @auth_middleware(
+        parser=usdc_transaction_history_parser,
+        id_param="id",
+        ns=full_user_ns,
```

Comment:
instead of passing the ns, we could just pass the abort method right?

```
@auth_middleware(
  abort=full_user_ns.abort
)
```

Reply:
Oh thank you, I don't know how I didn't think of that. I hated that I even had to pass the namespace....